site/content/contribute/server/developer-setup.md

```diff
@@ -3,6 +3,10 @@ title: "Developer Setup"
 date: 2017-08-20T11:35:32-04:00
 weight: 2
 subsection: Server
+markup:
+  goldmark:
+    renderer:
+      unsafe: true
```

Comment:
Even with this change `contribute/server/developer-setup/` is still not rendered correctly at my local instance. I'm using `Hugo Static Site Generator v0.62.2/extended linux/amd64 BuildDate: unknown`

I also tried using `defaultMarkdownHandler: blackFriday` to just continue using black friday, but that did work either.

Do you have ides why this doesn't work on my local instance?

Reply:
Hmm.  I'm not sure.  I thought I tested it in 0.62.2, but I'm not sure.  I'll try it with that specific version and let you know what I find.

In the interim, it looks like the reversion of the version* seems to have worked.

* Potential metal band name

Reply:
Realized the configuration needs to be set in `config.toml`, not in the top of the file.  Pushed some changes.